cmd/file_lint.go

```diff
@@ -184,6 +192,25 @@ func executeLint(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// LoadCustomFunctions will scan for (and load) custom functions defined as vacuum plugins.
+func LoadCustomFunctions(
+	functionsFlag string,
+	silence bool,
+) (map[string]model.RuleFunction, error) {
+	// check custom functions
+	if functionsFlag != "" {
+		pm, err := plugin.LoadFunctions(functionsFlag, silence)
+		if err != nil {
+			fmt.Printf("Error: Unable to open custom functions: %v\n", err)
+			fmt.Println()
+			return nil, err
+		}
+		fmt.Printf("Info: Loaded %d custom function(s) successfully.\n", pm.LoadedFunctionCount())
```

Comment:
do not directly print. Use `logbasics.Info` instead (output is then controlled by the `verbose` cli flag).